src/java/com/puppetlabs/puppetserver/JRubyPuppet.java

```diff
@@ -18,6 +18,7 @@
 public interface JRubyPuppet {
     Map getClassInfoForEnvironment(String environment);
     List getModuleInfoForEnvironment(String environment);
+    List getAllModuleInfoForEnvironment();
```

Comment:
Ah yeah, reading what I have there now doesn't really make sense. I'll update it.